specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -498,22 +596,21 @@ model ResourceMetric {
 
```

Comment:
why is id, resourcId, name, metricNamespace nullable?

Reply:
Updated them after discussion with Krishna